src/common/containers/__tests__/EditRecordAction.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { fromJS } from 'immutable';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { getStoreWithState } from '../../../fixtures/store';
+import EditRecordAction from '../EditRecordAction';
+
+describe('LoginOrUserDropdownContainer', () => {
+  it('renders edit button if user is cataloger or superuser', () => {
```

Comment:
Can you write 2 separate test cases, for`roles: ['cataloger']` and `roles: ['superuser']`